linkml_runtime/dumpers/csv_dumper.py

```diff
@@ -15,7 +16,7 @@
 
 class CSVDumper(Dumper):
 
-    def dumps(self, element: YAMLRoot,
+    def dumps(self, element: Union[BaseModel, YAMLRoot],
```

Comment:
There are two basic changes that happened in this PR, this is the first, where the YAMLRoot type was used, I changed to a union of BaseModel and YAMLRoot